public/controllers/RegisterController.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Descriere:
+ * Author: 
+ */
+
+var app = angular.module('contacts');
+app.controller('RegisterController', function($http, $state) {
+    var vm = this;
+
+    vm.user = {};
+
+    vm.days = [];
+    for (var i = 1; i < 32; i++) {
+        vm.days.push(i);
+    }
+    vm.years = [];
+    for(var i = 1900; i < 2017; i++) {
+        vm.years.push(i);
+    }
+    
+    $http.get('https://restcountries.eu/rest/v1/all').then(function (response) {
+      
+       vm.country = response.data;
+        
+      });
+    vm.checkPasswordValidity = function () {
+        vm.errorMessage = '';
+        if (vm.user.password !== vm.user.confirmPassword) {
+            vm.errorMessage = "Passwords don't match";
+        }
+       
+    };
+
+    vm.register = function () {
+
+        vm.errorMessage = '';
+
+        if (!vm.user.email) {
+            vm.errorMessage = 'Email is required';
+            return;
+        }
+        vm.errorMessage = '';
```

Comment:
Aici setezi `vm.errorMessage = '';`. Nu ai de ce. Nu strica la nimic chestia asta dar e in plus avand in vedere ca e deja setat la linia 36